addon/ui/panels.py

```diff
@@ -95,14 +95,19 @@ def draw(self, context):
             col = common.split_column(box)
             col.prop(model, 'prepend_armature')
             col.prop(model, 'ignore_transforms')
+            col.prop(model, 'transform_source')
 
-            align = col.column(align=True)
-            align.prop(model, 'origin_x', text='Origin X')
-            align.prop(model, 'origin_y', text='Y')
-            align.prop(model, 'origin_z', text='-Z')
+            if model.transform_source == 'OBJECT':
+                col.prop(model, 'transform_object')
+            else:
```

Comment:
I guess I'd probably put a newline before the `else` here, but that's just my preference, either way is fine.